index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="styles/header.css">
+  <link rel="stylesheet" href="styles/headline.css">
+  <link rel="stylesheet" href="styles/general.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="left-section">
+        <p class="logo">My Logo</p>
+      </div>
+      <button class="right-section">
```

Comment:
- Kindly make sure the logo (if the design has one) is on the left and has a link (a tag). I can suggest you use the `a ` instead of the `p `tag.
- [ ] Great job using the `<img>` tag here for your hamburger menu  However I think it would be better to change to also put it in the ` <nav>` tag, The nav tag is for semantics reasons. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info :+1: 

- [ ] Kindly make sure the logo (if the design has one) has a link (`a` tag).

* This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info. The logo (if the design has one) is on the left and has a link (a tag). :+1: 

example 
```
<header class="header"> 
  <div class="logo"> <a href="#"><img src="./images/logo.png" alt="Logo"></a></div>
   
      <nav class="menu">
        <a href="#"><img src="./images/hamburger.png" alt="hamburger" class="hamburger"></a>
      </nav>
    </header>
```

Reply:
I have used anchor tag for the logo.
I wrapped the icon in to navigation tag.
I added a link to the logo.
I have tried my best to address all of your valuable comments.

Thank you for the detail suggestions.